spec/requests/api/v1/passwords/create_spec.rb

```diff
@@ -1,55 +1,36 @@
-describe 'POST /api/v1/users/password', type: :request do
-  subject(:post_request) do
-    post user_password_path, params: params, as: :json
-  end
-
+describe 'POST /api/v1/users/password', { type: :request } do
```

Comment:
why are you adding brackets to the request type?

Reply:
[This](https://docs.rubocop.org/rubocop/1.3/cops_layout.html#enforcedlastargumenthashstyle-always_inspect-default) is the configuration that we have for hashes. We're allowing explicit and implicit ones. I prefer explicit hashes because I think it makes the code easier to read, so I always add braces 🤗